src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Dropdown.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import Icon from '../../components/Icon';
+import dropdownStyles from './dropdown.scss';
+
+export default class Dropdown extends React.PureComponent {
+    static defaultProps = {
+        isOpen: false,
+        onItemSelected: () => {}
```

Comment:
And I am not sure if I would make that public... Don't we always handle the click event on the item itself?